lib/msal-common/test/logger/Logger.spec.ts

```diff
@@ -187,6 +187,18 @@ describe("Logger.ts Class Unit Tests", () => {
             expect(logStore[LogLevel.Verbose]).toBe(undefined);
             expect(logStore[LogLevel.Trace]).toBe(undefined);
         });
+
+        it("Doesn't throw if loggerOptions explicitly set to undefined", () => {
+            //@ts-ignore
+            loggerOptions=void 0 || undefined || null;
```

Comment:
This will always evaluate to `null`, is the intent here to test `undefined`?

Reply:
thanks, will update

Reply:
updated to use undefined options only, thanks for pointing this out! :)